CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs

```diff
@@ -50,35 +55,35 @@ public async Task OnGet()
                 return;
           
             //Fetch user
-            var user = await _userManager.GetUserAsync(User);
-            
+            _user = await _userManager.GetUserAsync(User);
```

Comment:
Noot sure about moving this to a field... What if result is null, what if more functions are added (this is an index page so quite possible)?

Reply:
you suppose throwing this guy from function to function? 😄

Reply:
Yeah, I do. But I'm not sure... @KaterynaZorina ?

Reply:
Decided to add null check right after user setting

Reply:
@dennisahlin private field with null check looks ok for me. We can't guarantee that user won't pass `null` as a method parameter(unless we will be checking for null every time). `OnGet()` method will always be called, so we can rely on it.